modules/java/test/pure_test/build.xml

```diff
@@ -53,7 +54,7 @@
       <formatter type="xml"/>
 
       <batchtest fork="yes" todir="${test.dir}">
-        <zipfileset src="build/jar/opencv-test.jar" includes="**/${opencv.test.package}/${opencv.test.class}.class" excludes="**/OpenCVTest*">
+        <zipfileset src="build/jar/opencv-test.jar" includes="**/${opencv.test.package}/${opencv.test.class}.class" excludes="**/OpenCVTest*, ${opencv.test.filter}">
```

Comment:
this name would confuse anyone because it is used to exclude tests.

(AFAIK, original problem with QR test is already resolved)

Reply:
Replaced "filter" with "exclude".